### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by 214481859**
Chinese translation documents.

## Description

**PLEASE MAKE SURE THAT THE BRANCH PR INCLUDES JIRA TICKET ID** (_for centreon-internal_)

Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:
* **QA Team** (Quality Assurance) with tests.
* **reviewers** to understand what are the stakes of the pull request.

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [！ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 22.10.x
- [ ] 23.04.x
- [ ！] 23.10.x
- [ ] 24.04.x
- [ ] master

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by sc979 — CHANGES_REQUESTED** [review]
Thank you @214481859 for your contribution to the translation.
Could you please add the .po files instead of a zipfile ?

File: centreon/lang/zh_CN.UTF-8.zip
Comment (kduret) [inline centreon/lang/zh_CN.UTF-8.zip:1]: zip file must be replaced by `.po` file

**Review by kduret — CHANGES_REQUESTED** [review]
PR title must be updated

**Comment by 214481859** [comment]
Ok, I'm not familiar with the process.



---Original---
From: "Kevin ***@***.***&gt;
Date: Thu, Jul 4, 2024 15:46 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [centreon/centreon] Add files via upload (PR #4563)




 
@kduret requested changes on this pull request.
 
PR title must be updated

 
On centreon/lang/zh_CN.UTF-8.zip:
 
zip file must be replaced by .po file

**Pull request closed without merging**